src/WorkOS.net/Services/UserManagement/Enums/AuthenticationMethods.cs

```diff
@@ -0,0 +1,25 @@
+namespace WorkOS
+{
+    using System.Runtime.Serialization;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    /// <summary>
+    /// An enum describing the state of a <see cref="User"/>.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum AuthenticationMethods
```

Comment:
Enums should be named in the singular, as they can only be inhabited by a single value at a time:

```suggestion
    public enum AuthenticationMethod
```